CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+- Start ocamllsp server for DPM using dune tools exec ocamllsp (#1903)
```

Comment:
```suggestion
- Fix DPM (Dune Package Management) support for users with Dune installed via
  Opam. Previously, DPM support worked only when Dune was installed using
  the installation script for the Dune binary distribution. (#1903)
```

For the release notes, it's better to point out the user impact of the change than the technical change itself.